config/config.go

```diff
@@ -24,7 +25,10 @@ func LoadFile(filename string) (*Config, error) {
 
 var (
 	DefaultModule = Module{
-		Version: 2,
+		Version:        2,
+		Retries:        3,
+		MaxRepititions: 25,
+		Timeout:        60,
```

Comment:
> We should drop this to a second as it turns out to be a per-query timeout. 

You should first review my comments on `collector.go` lines 42-43. If you would like the change described there then yes what you say makes more sense though I would suggest 20 seconds as the current hard coded values called for 3 retries in 60 seconds.

> Shouldn't this be a time.Duration?

In the struct I specified it as a `time.Duration` which is a, pardon the non-Go description, renaming of int64. As I said I'm not a Go expert. I had initially attempted to use `int` but found that Go did not want to do automatic type casting since it was not a constant (60 is a constant so I assume that the compiler automatically does the type cast at compilation time). To me the more natural use of a timeout is to specify it in seconds. Granted it could be done in milliseconds or nanoseconds (nanoseconds is the smallest increment of `time.Duration`) but I would expect most people would not expect an increment of less than a second on a request/response connection like this. For your convenience: [see here](https://golang.org/pkg/time/#Duration).